pages/base.page.js

```diff
@@ -0,0 +1,44 @@
+const PAGE_TIMEOUT = 10000
+export default class BasePage{
+    
+    /**
+     * Abrir pagina
+     * @param{String} ruta a la cual acceder
+     * 
+     */
+
+    async open(url) {
+
+        await browser.url(url);
+
+    }
+
+    /**
+     * Esperar a que un elemento sea clickeble y hacer click
+     * @param{Object} elemento a clickear
+     */
+
+    async clickElement(element){
+
+        await element.waitForClickable({ timeout: PAGE_TIMEOUT });
+        await element.click();
+
+    }
+
+    /**
+     * Metodo para enviar texto a un elemento
+     * @param{Object} elemento que recibira el texto
+     * @param{String} texto a enviar
+     * 
+     */
+
+    async emptyFieldAndSendText(element, text){
+
+        await element.waitForClickable({ timeout: PAGE_TIMEOUT });
+        await element.clearValue();
+        await element.click();
+        await element.keys(text);
```

Comment:
If you use the `setValue()` method wdio provides for web elements, you can avoid much of this logic, simplifying method to just:

```javascript
async emptyFieldAndSendText(element, text){
     await element.waitForClickable({ timeout: PAGE_TIMEOUT });
     await element.setValue(text);
}
```